bitnami/drupal/10/debian-11/rootfs/opt/bitnami/scripts/libdrupal.sh

```diff
@@ -127,6 +127,11 @@ drupal_initialize() {
         fi
     fi
 
+    # Check if we should install Drupal dependencies with Composer
+    if is_boolean_yes "${DRUPAL_COMPOSER_INSTALL:-}"; then
+        drupal_composer_install
+    fi
+
```

Comment:
this could be changed into:
```
is_boolean_yes "$DRUPAL_COMPOSER_INSTALL" && drupal_compose_install
```

and `DRUPAL_COMPOSER_INSTALL` (and related env vars) values should be checked in the `drupal_validate` function.